src/main/java/frc/robot/RobotContainer.java

```diff
@@ -150,29 +148,18 @@ public Command getAutonomousCommand() {
     new RamseteCommand(
         trajectory,
         driveBaseSubsystem::getPose,
-        new RamseteController(Constants.ramseteB, Constants.ramseteZeta), //Fix these constants by
-                                                                            //characterizing the robot
-        new SimpleMotorFeedforward(
-            Constants.sVolts,
-            Constants.vVoltSecondsPerMeter,
-            Constants.aVoltSecondsSquaredPerMeter),
-
+        new RamseteController(Constants.ramseteB, Constants.ramseteZeta), // ramsete follower to follow trajectory
         Constants.driveKinematics,
-        
-        driveBaseSubsystem::getWheelSpeeds,
-        new PIDController(1, 0, 0),
-        new PIDController(1, 0, 0),
-        //RamseteCommand passes volts to the callback
-        driveBaseSubsystem::setAutonVolts,
+        driveBaseSubsystem::acceptWheelSpeeds,
         driveBaseSubsystem);
```

Comment:
None of the SysId values are used with this constructor. If it works, great, but it seems like it defeats the purpose of the characterization.